docs/pages/eas-insights/index.js

```diff
@@ -0,0 +1,3 @@
+import redirect from '~/common/redirect';
+
+export default redirect('/eas-update/introduction');
```

Comment:
This should be `insights` instead of `update`?
```suggestion
export default redirect('/eas-insights/introduction');
```